src/renderer/services/neo.js

```diff
@@ -707,8 +727,8 @@ export default {
             balance: token.balance,
           });
         })
-        .catch(() => {
-          resolve({ balance: 0 });
+        .catch((ex) => {
+          reject(ex);
```

Comment:
Can we make this an explicit error?

Reply:
the error doesn't go back to the user on this one.